plugins/modules/mount.py

```diff
@@ -250,9 +253,9 @@ def mount(module):
         if mount_dir or mount_over_dir:
             if is_fspath_mounted(module, mount_dir, mount_over_dir):
                 if mount_dir:
-                    result['msg'] = "Filesystem/Mount point '%s' already mounted" % mount_dir
+                    result['msg'] = "Filesystem/Mount point '%s' already unmounted" % mount_dir
```

Comment:
why has "mounted" been changed to "unmounted" on these lines? aren't we checking if something has already been mounted? I've never seen a "this mount point has already been unmounted" message before

Reply:
yes you are right. i meant to change the one umount function instead of the mount function.
i will be changing that. thanks for catching it

Reply:
i have made the msg changes on the correct place now.